bindings/jupyter-modules/jupyter-ma-causal/src/selectors/base-selectors.js

```diff
@@ -0,0 +1,28 @@
+import {createSelector} from 'reselect';
+import {CONTAINER_PADDING} from '../constants';
+
+export const rootSelector = state => state;
+
+export const getContainerWidth = createSelector(
+  rootSelector,
+  state => state.containerWidth
+);
+
+export const getContainerHeight = createSelector(
+  rootSelector,
+  state => state.containerHeight
+);
+
+export const getColumnWidth = createSelector(rootSelector, () => 300);
```

Comment:
nit: make the width a constant or let's add TODOs for items you're aware that we need to change after the scaffold.

Reply:
Addressed.